book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] You have done great creating the `add_rental` method as suggested by the previous reviewer. But your current implementation is not correct. the method is meant to take 2 parameters and then create an instance of `Rental` which you can then push into the `@rentals` array. :+1: :100: I suggest you update this. Here's a snippet of possible implementation. :+1:  

```ruby
  def add_rental(person, date)
   @rentals <<  Rental.new(date, self, person)
  end
```